django_redis/client/default.py

```diff
@@ -540,3 +540,19 @@ def close(self, **kwargs):
                 for c in self._clients[i].connection_pool._available_connections:
                     c.disconnect()
                 self._clients[i] = None
+
+    def touch(self, key, timeout=DEFAULT_TIMEOUT, version=None, client=None):
+        """
+        Sets a new expiration for a key.
+        """
+
+        if client is None:
+            client = self.get_client(write=True)
+
+        key = self.make_key(key, version=version)
+
+        if client.exists(key):
```

Comment:
Can we drop this check? My understanding is `expire` will ignore keys that doesn't exist.

Could use a test for the key-doesn't-exist case.